src/actions/build/compiler_message_parser.rs

```diff
@@ -0,0 +1,96 @@
+use std::path::PathBuf;

```

Comment:
Again, not your fault, but for easier review, could you separate the moving of code to another file into a separate commit from any changes?